### PR TITLE
Fix groupware visual issue and improve spacing

### PR DESCRIPTION
**Pull request by quentinguidee**
## Motivation

- There's a bug in the groupware > availability section where the text is collapsed in the table.
- The spacing of the page doesn't allows to instantly see the difference between the description, the settings and the save button.

## Screenshots

<details>
  <summary>Before</summary>
  screenshot-before
</details>

<details>
  <summary>After collapsed text fix</summary>
  screenshot-after-1
</details>

<details>
  <summary>After spacing rework</summary>
  
  screenshot-after-2
  screenshot-after-3
</details>

### CODE REVIEW

**Comment by quentinguidee** [comment]
(not reflected in the screenshots, but the last commit lowers the "+" icon of 1px)

File: apps/dav/src/views/Availability.vue [hunk 1]
```diff
@@ -49,6 +49,7 @@
 				<button :key="`add-slot-${day.id}`"
 					:disabled="loading"
 					class="icon-add add-another button"
+					:class="{ 'add-another-not-first': day.slots.length !== 0 }"
```
Comment (CarlSchwan) [inline apps/dav/src/views/Availability.vue:52]: Is this new class used?

Comment (quentinguidee) [reply apps/dav/src/views/Availability.vue:52]: Yes ; In fact, when there is no slot, the row is dense (see screenshot in purple), but when a slot is added, a bottom padding is added to the center box (not to be confused with events of the next day, see screen of the original message). Therefore, the + symbol is too low and is no longer aligned with the last slot, making the role of the "+" symbol less clear.

This class therefore makes it possible to know whether the "+" icon should be raised or not.
However, If adding a new class is disturbing and someone sees another solution, I can change that.

image

Comment (quentinguidee) [reply apps/dav/src/views/Availability.vue:52]: Ok I think I found a better solution… Will try that

**Comment by quentinguidee** [comment]
Screenshots after the last commit to remove additional class :

Capture d’écran 2021-12-20 à 12 08 24

Capture d’écran 2021-12-20 à 12 08 32

**Review by jancborchardt — APPROVED** [review]
Looks nice, thank you @quentinguidee! :)

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**